### DISCUSSION TITLE
Reuse a media file

### DISCUSSION THREAD

**Discussion by Tarocch1** [anyproto/contributors#74] [Anytype apps]
I noticed that Anytype has some built-in media file types, such as image、audio and video. If I upload an image in the editor, this image will be automatically categorized as an "Image" type.

If I upload and use an image on one page, it can be quite cumbersome to display the same image on another page：I enter "/image", then I can paste a link or upload an image, but I can't select an image that's already in the image library. This is strange. It would be more convenient if there was an option to quickly reuse an already uploaded media file.

**Comment by fuksman** [comment]
Hello @Tarocch1, thank you for bringing attention to this. We plan to address this issue by implementing files as regular objects. 
This will make it possible to reuse them, as well as add descriptions, tags, and other relations.
It is difficult to provide specific dates as we are currently focused on another aspect of our roadmap.
